packages/cli/src/project.ts

```diff
@@ -24,7 +24,8 @@ export class Project extends Item {
     let config = this._config.___experimentalFlags_WILL_CHANGE_IN_PATCH || {};
     return {
       newEntrypoints: !!config.newEntrypoints,
-      useSourceInsteadOfGeneratingTSDeclarations: !!config.useSourceInsteadOfGeneratingTSDeclarations
+      useSourceInsteadOfGeneratingTSDeclarations: !!config.useSourceInsteadOfGeneratingTSDeclarations,
+      useTSMorphToGenerateTSDeclarations: !!config.useTSMorphToGenerateTSDeclarations,
```

Comment:
is the plan to remove `useSourceInsteadOfGeneratingTSDeclarations` now then?

Reply:
yep